README.md

```diff
@@ -90,21 +128,24 @@ and the role will install `1:7.16.0-1` on Debian- and SUSE-based systems, `7.16.
 
 ## Agent 5 (older version)
 
-This role includes support for Datadog Agent version 5 for linux only. To install Agent 5, you need to:
+This role includes support for Datadog Agent version 5 for Linux only. To install Agent 5, you need to:
 
-1. Set `datadog_agent5` parmeter to `true`.
+1. Set `datadog_agent_major_version` parmeter to `5`.
 2. Set `datadog_agent_version` to an existing Agent v5 version or leave it empty to always install the latest version (`5.*`).
 
-To downgrade from Agent v6 to Agent v5, you need to (**on centos this will only work with ansible 2.4 and up**):
+## Agent major version downgrades
```

Comment:
We could merge the both paragraph since the role is much simpler now.

Something like:
```
1 .Either set `datadog_agent_major_version` to the major version you want to install the latest of that version or set `datadog_agent_version` to a specific version to (ex: 5.32.5)
2. Set `datadog_agent_allow_downgrade` to `yes`.
```

WDYT ?

Reply:
👍, moreover since `datadog_agent_major_version` is not even needed if `datadog_agent_version` is set, we can remove the bit about `datadog_agent_major_version`:
 
```
1. Set `datadog_agent_version` to a specific version to downgrade to (ex: 5.32.5)
2. Set `datadog_agent_allow_downgrade` to `yes`.
```

Setting only `datadog_agent_allow_downgrade` and `datadog_agent_major_version` does not work, you need to specify a minor version for the downgrade to work.